lapis2-docs/tests/docs.spec.ts

```diff
@@ -55,13 +107,16 @@ const pages = [
 ];
 
 test.describe('The documentation', () => {
-    test('should show all expected pages via next buttons and all relative links should work', async ({ page }) => {
-        await page.goto(baseUrl);
+    pages.forEach((documentationPage) => {
```

Comment:
```suggestion
    pages.forEach((documentationPage, indexOfCurrentPage) => {
```
and pass that to `findNeighbouringPages`, which saves us the weird computation there.